### DISCUSSION TITLE
Offline

### DISCUSSION THREAD

**Discussion by kylenstone** [excalidraw/excalidraw#6767] [Q&A]
What is the latest (2023) option for running Excalidraw offline?  Asking as I'm about to board a plane!

**Comment by ad1992** [comment]
@kylenstone Excalidraw supports PWA so you install the app and use if offline 🙂

![image](https://github.com/excalidraw/excalidraw/assets/11256141/78eea634-4000-4f5c-a5ca-467093a032ed)

**Comment by LBF38** [comment]
@kylenstone  You can also use it with [Obsidian.md](https://obsidian.md) and the Excalidraw plugin for a super UX, all offline.
